install/installer/traf_apache_mods

```diff
@@ -0,0 +1,267 @@
+    $PDSH_HADOOP_NODES sudo rm -rf  $HBASE_HOME/lib/hbase-trx* 2>/dev/null
+    $TRAF_PDSH mkdir -p $LOCAL_WORKDIR 2>/dev/null
+    $PDSH_HADOOP_NODES mkdir -p $LOCAL_WORKDIR 2>/dev/null
+    cp $UNTAR_DIR/export/lib/$hbase_trx_jar $LOCAL_WORKDIR
+    cp $UNTAR_DIR/export/lib/$traf_util_jar $LOCAL_WORKDIR
+    $PDCP_HADOOP_NODES $LOCAL_WORKDIR/$hbase_trx_jar $LOCAL_WORKDIR
+    $PDCP_HADOOP_NODES $LOCAL_WORKDIR/$traf_util_jar $LOCAL_WORKDIR
+    $PDSH_HADOOP_NODES sudo cp $LOCAL_WORKDIR/$traf_util_jar $HBASE_HOME/lib
+    $PDSH_HADOOP_NODES sudo cp $LOCAL_WORKDIR/$hbase_trx_jar $HBASE_HOME/lib
+    $PDSH_HADOOP_NODES sudo chmod 644 $HBASE_HOME/lib/$hbase_trx_jar
+    $PDSH_HADOOP_NODES sudo chmod 644 $HBASE_HOME/lib/$traf_util_jar
+
+    $PDSH_HADOOP_NODES rm $LOCAL_WORKDIR/$hbase_trx_jar 2>/dev/null
+    $PDSH_HADOOP_NODES rm $LOCAL_WORKDIR/$traf_util_jar 2>/dev/null
+else
+    for node in $HBASE_NODES
+    do 
+    ssh -q -n $node sudo rm -rf $HBASE_HOME/lib/hbase-trx* 2>/dev/null
+    ssh -q -n $node sudo mkdir -p $TRAF_WORKDIR 2>/dev/null
+    ssh -q -n $node sudo chmod 777 $TRAF_WORKDIR
+    scp -q $UNTAR_DIR/export/lib/$hbase_trx_jar $(whoami)@$node:$TRAF_WORKDIR
+    scp -q $UNTAR_DIR/export/lib/$traf_util_jar $(whoami)@$node:$TRAF_WORKDIR
+    ssh -q -n $node sudo cp $TRAF_WORKDIR/$hbase_trx_jar $HBASE_HOME/lib
+    ssh -q -n $node sudo cp $TRAF_WORKDIR/$traf_util_jar $HBASE_HOME/lib
+    ssh -q -n $node sudo chmod 644 $HADOOP_PATH/$hbase_trx_jar
+    ssh -q -n $node sudo chmod 644 $HADOOP_PATH/$traf_util_jar
+    done
+fi
+
+#=======================================
+#Check that HBase-trx copied to all nodes
+
+for node in $HBASE_NODES
+do
+   copiedOver=$(ssh -q -n $node sudo ls $HBASE_HOME/lib/hbase-trx* | wc -l)
+   if [[ $copiedOver -ne "1" ]]; then
+      echo "***ERROR: $hbase_trx_jar was not copied on $node"
+      echo "***ERROR: Please investigate why this happened"
+      echo "***ERROR: Trafodion can not start without this. EXITING..."
+      exit -1
+   fi
+done
+
+echo "***INFO: $hbase_trx_jar copied correctly! Huzzah."
+
+
+
+#Copy hbase-site.xml file
+ssh -q -n $HBASE_NODE sudo cp $HBASE_HOME/conf/hbase-site.xml $HOME
+ssh -q -n $HBASE_NODE sudo chown $(whoami).$(whoami) $HOME/hbase-site.xml
+ssh -q -n $HBASE_NODE sudo chmod 777 $HOME/hbase-site.xml
+
+scp -q $(whoami)@$HBASE_NODE:$HOME/hbase-site.xml $HOME
+if [[ $? -gt 1 ]]; then
+   echo "***ERROR: Unable to find $HBASE_HOME/conf/hbase-site.xml file on $HBASE_NODE or unable to copy."
+   exit -1
+fi
+sudo cp $HOME/hbase-site.xml $TRAF_WORKDIR
+sudo chown trafodion.trafodion $TRAF_WORKDIR/hbase-site.xml
+
+#=====================================
+# create new directories for bulkload and lobs if not already there
+rm $LOCAL_WORKDIR/traf_temp_output 2>/dev/null
+
+ssh -q -n $HDFS_NODE 'sudo su' "$HDFS_USER" '--command "' "$HADOOP_PREFIX"'/bin/hdfs dfs -mkdir /hbase-staging" 2> $HOME/traf_temp_output'
+if [ $? != 0 ]; then
+   # ok if directory already exists
```

Comment:
If you use dfs -mkdir -p otion, I believe it will return zero exit if file exists.